### ISSUE TITLE
Printing support for binary format of numbers inside Taichi scope

### ISSUE THREAD

**Issue by Linus-Civil** [taichi-dev/taichi#5972]
**Describe of the proposed feature**
The bitwise operations of numbers are often performed in computer programs, such as the parallel radix sorting based on GPU.  In this case, I need to verify that each bitwise operation has achieved the desired results. Therefore, I want to print the binary format of the manipulated number directly in Taichi scope (kernel and func). 

**Describe the solution you'd like (if any)**
- For unsigned integers (ti.uint32), I hope to get output of `0100 0000 0100 1000 1111 0101 1100 0011` after executing `a=ti.uint32(1078523331); print(bin(a))` in Taichi scope.
- For signed integers (ti.int32),  I hope to get output of `1111 1111 1111 1111 1111 1111 1111 0110`  (two's complement) after executing `b=ti.int32(-10); print(bin(b))` in Taichi scope.

**Additional comments**
To sum up, I hope that the interface function `bin()` can get all bits in the memory of target numbers.  As far as my requirements are concerned, interface functions need to support both unsigned and signed integers.

**Comment by yuanming-hu** [comment]
Given Taichi doesn't support strings, the optimal way sounds like using f-strings:

```python
@ti.kernel
def foo():
    x = 123
    print(f'{x:b}')
```

Related issue: #5949

**Comment by Linus-Civil** [comment]
> Given Taichi doesn't support strings, the optimal way sounds like using f-strings:
> 
> ```python
> @ti.kernel
> def foo():
>     x = 123
>     print(f'{x:b}')
> ```
> 
> Related issue: #5949

Anyway, just choose an easier way of development.

**Comment by ailzhang** [comment]
This will require some tweaks to generate the format string in binary form IIUC, adding it to todo queue for now.
